config.js

```diff
@@ -543,6 +545,16 @@ config.LOG_REPLICATION_ENABLED = true;
 config.AWS_LOG_CANDIDATES_LIMIT = 10;
 config.BUCKET_LOG_REPLICATOR_DELAY = 5 * 60 * 1000;
 
+///////////////////////////
+//      KEY ROTATOR      //
+///////////////////////////
+
+config.KEY_ROTATOR = {
```

Comment:
don't use an object in `config.js`. we can't override it with env vars. for flat configs we can override it by setting for example `CONFIG_JS_KEY_ROTATOR_ENABLED=false`